compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -1183,6 +1183,7 @@ static void buildLLVMGPUCodegenConfigurationPassPipelineImpl(
     funcPassManager.addPass(createConfigTrackingCanonicalizerPass);
     funcPassManager.addPass(createCSEPass);
   }
+  modulePassManager.addPass(createMaterializeTuningSpecsPass());
```

Comment:
I think the pass ordering might be off... Do we want to do user configs first and then tuning spec. User config should be the ultimate override.

Reply:
User configs do come first. It's documented in the materialize user configs pass. Materialize tuning configs doesn't run the transform dialect interpreter and it's up to the materialize user configs to decide which transforms library to run.

Reply:
Maybe I need to come up with a different name to better disambiguate these two passes...